src/bug_report.rs

```diff
@@ -106,13 +82,22 @@ fn format_env_info(starship_version: &str, environment: Environment) -> String {
         starship_version = starship_version,
         shell_name = environment.shell_info.name,
         shell_version = environment.shell_info.version,
-        os_name = environment.os_type,
-        os_version = environment.os_version,
         terminal_name = environment.terminal_info.name,
         terminal_version = environment.terminal_info.version,
+        os_name = environment.os_type,
+        os_version = environment.os_version,
         shell_config = environment.shell_info.config,
         starship_config = environment.starship_config,
+    ))
+        .replace("%20", "+");
+
+    format!(
+        "https://github.com/starship/starship/issues/new?body={}",
+        body
     )
+    .chars()
+    .take(8100) // Magic number accepted by Github
```

Comment:
Make 8100 a `const`, although I'm not sure just cutting off everything past 8100 chars is the right way to do this

Reply:
That makes two of us, but I figured it's better than making Github throw an error.